sqlite3/lib/src/wasm/js_interop.dart

```diff
@@ -34,6 +34,24 @@ extension JsContext on _JsContext {
   external IdbFactory? get indexedDB;
 }
 
+extension IdbFactoryExt on IdbFactory {
+  Future<List<DatabaseName>> databases() async {
+    final jsDatabases = await promiseToFutureAsMap(
+        callMethod<Object>(this, 'databases', const []));
```

Comment:
In general, I try to avoid `js_util`, can we use `@staticInterop` instead? `@staticInterop` isn't documented that well, I can also help with the migration if you want me to.